docs/cloud/serverless.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
Filename should be cloud-serverless.md
It should go into the /docs/cloud/cloud-databases folder because that's what it's related to.